### PR TITLE
[Cult 3.0] Path Entrance, Path Exit & Pulse, Runes XVI, XVII & XVIII

### PR DESCRIPTION
**Pull request by DeityLink**
path

=Path Entrance (Travel, Self, Other)=
* When used, the player must choose a "path", among the 10 cult words.
* Once the path is set, the rune takes the appearance of a small altar.
* When used again, everything on top of the rune will be jaunted after a second to the path's Exit.
* If no Exit exist, no jaunt occurs
* Can be attuned to talismans to remotely trigger the jaunt
* Erasing a word of the rune will allow you to set a different path

=Path Exit (Travel, Other, Self)=
* When used, the player must choose a "path", among the 10 cult words, that hasn't been used by another Exit.
* Once the path is set, the rune takes the appearance of a small altar.
* Serves as a destination for Path Entrance runes
* Can be attuned to talismans to instantly jaunt there
* An inactive Path Exit rune may be imbued into a talisman allowing you to choose an available path
* Using the talisman a second time will then trigger the jaunt.

=Pulse=
* Basically the EMP rune, ported as is without any fancy extra effects, might refine that later.

### CODE REVIEW

**Comment by DeityLink** [comment]
Talisman sprite updates with the attuned Path rune's word: 
image

You can see it pulse brighter than the other words.
It's very subtle but should be enough to differentiate two talismans teleporting to different paths.
